dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -241,15 +253,23 @@ def build_record(self, ts: datetime, source: Path, service: str, pid: int, messa
             ("string", "message"),
         ]
 
-        record_values = {}
-        record_values["ts"] = ts
-        record_values["source"] = source
-        record_values["service"] = service
-        record_values["pid"] = pid
-        record_values["message"] = message
-        record_values["_target"] = self.target
-
-        for key, value in self._parse_additional_fields(service, message).items():
+        record_values = {
+            "ts": ts,
+            "message": line,
+            "service": None,
+            "pid": None,
+            "source": source,
+            "_target": self.target,
+        }
+
+        match = RE_LINE.match(line)
+        if match:
+            values = match.groupdict()
+            del values["ts"]
```

Comment:
I understand, but that's for the purpose of matching the timestamp. If I'm correct, you don't need to capture the timestamp here, because you're deleting it anyway.

In that case, I would recommend to look at the two code suggestions I've done further down. In both cases, the timestamp is not "captured" but it is matched and therefore you don't need the del action here